templates/ovndbcluster/bin/setup.sh

```diff
@@ -33,8 +38,9 @@ else
     DB_ADDR="[::]"
 fi
 
+# TODO: this logic is dedicated for non-leaders. The assumption that -0 is
```

Comment:
It was done for initial bootstrap(add *remote* args to non-0 pod), later cluster leader movement should work and not consider -0 pod as leader.

Reply:
Will update the comments to reflect the reasoning.